src/common/containers/content/pages/widgets/data_tables/table_entries/ProjectsDataTableEntry.jsx

```diff
@@ -9,8 +9,12 @@ import { connect } from 'react-redux';
 import ProjectsDataTableEntry from '../../../../../../components/content/widgets/data_tables/table_entries/ProjectsDataTableEntry'; // eslint-disable-line max-len
 
 const mapStateToProps = (state) => {
+    const { basePath } = state;
+    const { user } = state.user;
```

Comment:
I think it'll just pull the initial user state from the [default reducer state](https://github.com/webgme/user-management-page/blob/projects-by-owner-page/src/common/reducers/users.js#L8)